adapters/aidem/aidem.go

```diff
@@ -0,0 +1,98 @@
+package aidem
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
```

Comment:
nitpick - you do not have to declare an error var here. In the below code, 
On line 25, You could simply do like - `return nil, []error{err}`
and on line 36, you could return `nil` instead of `errs`

Reply:
@gargcreation1992 fixed in latest commit.